core/templates/pages/classrooms-page/classrooms-page-root.component.html

```diff
@@ -0,0 +1,9 @@
+<oppia-angular-root>
```

Comment:
Why do you need this component? What if you merge it with the classrooms-page?

Reply:
It initializes some services. Yes we can put all these components to classrooms page too and completely remove the root component also but I don't think we should do that as that will break the existing pattern.

Reply:
Consistency is a good argument, let's keep it then